articles/manual/statements.md

```diff
@@ -0,0 +1,928 @@
+namespace has been opened earlier in the current block. 
+As mentioned above, even if the `X` namespace has been opened, it is not possible to reference the operation as `Y.Op`.
+If a short name `Z` for `X.Y` has been defined in that namespace and file, then `Op` needs to be referred to as `Z.Op`. 
+
+```qsharp
+namespace NS {
+    open Microsoft.Quantum.Intrinsic; // opens the namespace
+    open Microsoft.Quantum.Math as Math; // defines a short name for the namespace
+}
+```
+
+It is usually better to include a namespace by using an `open` directive.
+Using a fully-qualified name is required if two namespaces define constructs
+with the same name, and the current source uses constructs from both.
+
+## Formatting
+
+We are working to ensure that common punctuation marks are used consistently in all situations.
+We expect that this will make Q# easier to learn and to read because these marks always mean the same thing, and therefore the same concept is always represented the same way.
+
+Specifically:
+
+- The semi-colon, `;`, is used to end a statement or single-line directive.
+	It is not used for any other purpose.
+- The comma, `,`, is used to separate elements of a sequence. It is used for tuple literals, array literals, argument lists, tuple definitions, and type lists. 
+	**In a change from earlier versions, `;` is no longer supported as an array literal separator.**
+- The colon, `:`, is used to introduce a type annotation. It is primarily used in callable signatures.
+	Because a colon always introduces a type signature, the ternary conditional operator introduced in 0.3 uses a vertical bar, `|`, to separate the true and false values; thus, Q# uses `cond ? tval | fval` instead of the colon as separator as in C.
+
+Statements and declarations such as `for` and `operation` that end with a statement block do not require a terminating semicolon.
+Each statement description notes whether the terminating semicolon is required.
+
+Statements, like expressions, declarations, and directives, may be broken out across multiple lines.
+Having multiple statements on a single line should be avoided.
+
+## Statement Blocks
+
+Q# statements are grouped into statement blocks.
+A statement block starts with an opening `{` and ends with a closing `}`.
+
+A statement block that is lexically enclosed within another block is considered to be a sub-block of the containing block; containing and sub-blocks are also called outer and inner blocks.
+
+## Local Variables: Symbol Binding and Assignment
+
+Q# distinguishes between mutable and immutable symbols.
+
+In general, the use of immutable symbols is encouraged because it allows the compiler to perform more optimizations.
+
+The left-hand-side of the binding consists of a symbol tuple, and the right hand side of an expression.
+
+Since all Q# types are value types - with the qubits taking a somewhat special role - formally a "copy" is created when a value is bound to a symbol, or when a symbol is rebound. 
```

Comment:
```suggestion
Since all Q# types are value types — with qubits taking a somewhat special role — formally a "copy" is created when a value is bound to a symbol, or when a symbol is rebound. 
```